imagine/README.md

```diff
@@ -0,0 +1,74 @@
+# Pilosa-load: A load test for Pilosa
+
+This is a sort of placeholder for a higher level end-to-end
+test that could be pointed at an instance of the Pilosa database
+(github.com/pilosa/pilosa) and do some testing on it. This is loosely
+inspired by things like the Star Schema Benchmark, with the idea being to
+create a tool to let similar tests be run on different scales, against
+an arbitrary install.
+
+## Goals
+
+The basic goal here is to be able to construct a data set to do experiments
+on, and then do those experiments, in a controlled fashion. To that end, we
+describe a couple of indexes, with scalable parameters.
+
+One of the motivations for this is exploring performance of cross-index
+queries. To that end, one of the indexes created has a BSI field which
+corresponds to a column in another index. This is a plausible real-world use
+case which is nigh pessimal for Pilosa's current architecture.
+
+In the longer term, it'd be nice to be able to, say, turn CPU profiling
+on or off for a server corresponding to phases of a test. That might work
+better as a separate tool which invokes this one.
+
+## Indexes
+
+All indexes used by this tool have names starting with `imaginary-`. The
+intent here is that it should be safe to delete these indexes after tests,
+but just to be careful, we delete indexes only if we created them.
+
+In the interests of visualization, we imagine a system which has data about
+various entities, called `users`, and things that might happen involving
+those users, called `events`. Benchmarks might well operate on only one of
+these indexes, but having three of them lets us do interesting comparisons.
```

Comment:
three?

Reply:
yes, there's been exactly one commit and the readme is already bitrotted. :P